.travis.yml

```diff
@@ -73,8 +73,8 @@ env:
     - CIRCLE_NODE_INDEX=-1  # Avoid accidentially being a CircleCI worker
     - USE_PPAS="marutter/rdev staticfloat/juliareleases ondrej/golang"
     - R_LIB_USER=~/R/Library
-    - LINTR_COMMENT_BOT=false
-    - PATH="/opt/cabal/1.24/bin:$PATH:$TRAVIS_BUILD_DIR/node_modules/.bin:$TRAVIS_BUILD_DIR/vendor/bin:$HOME/dart-sdk/bin:$HOME/.cabal/bin:$HOME/infer-linux64-v0.7.0/infer/bin:$HOME/pmd-bin-5.4.1/bin:$HOME/bakalint-0.4.0:$HOME/elm-format-0.18:$HOME/.local/tailor/tailor-latest/bin:$HOME/phpmd"
+    - LINTR_COMMENT_BOT=falses
```

Comment:
I am not really sure what happens here. It is written false on my system and it is not the first time this is happening